.circleci/config.yml

```diff
@@ -1,15 +1,15 @@
 version: 2.1
 
 orbs:
-  rok8s: reactiveops/rok8s-scripts@8.1.0
+  rok8s: fairwinds/rok8s-scripts@8.1.0
 
 workflows:
   build_and_push:
     jobs:
       - rok8s/docker_build_and_push:
           config_file: deploy/build.config
           docker-login: true
-          username: "fairwinds+circleci"
+          username: "reactiveops+circleci"
```

Comment:
This shouldn't be changed back.  I updated this repo already to use the new one.